DEVELOPER_GUIDE.md

```diff
@@ -22,15 +22,12 @@ another JDK installation, e.g. `RUNTIME_JAVA_HOME=/usr/lib/jvm/jdk-8`.
 
 1. Change your node version to the version specified in `.node-version` inside the OpenSearch Dashboards root directory.
 1. Create a `plugins` directory inside the OpenSearch Dashboards source code directory, if `plugins` directory doesn't exist.
-1. Check out this package from version control into the `plugins` directory.
+1. Check out this package from version control into the `plugins/dashboards-reporting` directory.
    ```
-   git clone git@github.com:opensearch-project/dashboards-reports.git plugins --no-checkout
-   cd plugins
-   echo 'dashboards-reports/*' >> .git/info/sparse-checkout
-   git config core.sparseCheckout true
-   git checkout dev
+   git clone https://github.com/opensearch-project/dashboards-reporting.git plugins/dashboards-reporting
+   git checkout main
```

Comment:
not sure if it's intentional but the first command clones dashboards-reporting, the second command checks out `main` of dashboards core. also checkout main of reporting is not necessary since it's default

maybe 
```bash
cd plugins
git clone https://github.com/opensearch-project/dashboards-reporting.git
```

Reply:
Thanks for catching this. It was not intentional. Please check https://github.com/opensearch-project/dashboards-reporting/pull/18